kodilla-testing/src/test/java/com/kodilla/testing/shape/ShapeCollectorTestSuite.java

```diff
@@ -0,0 +1,68 @@
+package com.kodilla.testing.shape;
+
+import com.kodilla.testing.com.kodilla.testing.shape.*;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ShapeCollectorTestSuite {
+private static int counter=1;
+
+    @Before
+    public void Before (){
+        System.out.println("Start testu nr " + counter );
+    }
+    @After
+    public void After(){
+        System.out.println("Koniec testu nr " + counter );
+        counter++;
+    }
+
+    @Test
+    public void testAddFigure(){
+//given
+        ShapeCollector objShapeCollector = new ShapeCollector();
+//when
+        objShapeCollector.addFigure(new Circle(2.0));
+//then
+        Assert.assertEquals(1,objShapeCollector.shapesList.size());
+    }
+
+    @Test
+    public void testRemoveFigure() {
+//given
```

Comment:
you test should also cover situation when you are trying to remove figure which was not added to ShapeCollector